src/patternfly/components/Select/examples/Select.md

```diff
@@ -445,14 +445,14 @@ The plain select variation should be used when you do not want a border applied
 
 ### Placeholder item disabled
 ```hbs
-{{#> select id="select-placeholder-expanded" select--IsExpanded="true" select--IsPlaceholder="true" select--HasPlaceholderItem="true" select--PlaceholderItemDisabled="true"}}
+{{#> select id="select-placeholder-item-disabled" select--IsExpanded="true" select--IsPlaceholder="true" select--HasPlaceholderItem="true" select--PlaceholderItemDisabled="true"}}
```

Comment:
This is out of scope, but it would be nice to replace `id` with `select--id` in the hbs for this component. If you'd like to do that, feel free to create a separate issue. It could be a lot to update though since we have selects in a lot of our examples and demos.

Reply:
Created https://github.com/patternfly/patternfly/issues/4999 as a follow-up to address this.